browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -381,6 +381,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			spinfield.setAttribute('disabled', 'true');
 		}
 
+		spinfield.setAttribute('aria-disabled', Boolean(data.enabled));
```

Comment:
wrong logic:     should be `data.enabled === false` (if not present it means it is enabled too)

Reply:
Hmm, it is a core property of every widget, it should always exist. Can you give some example to investigate?

Reply:
disabled != enabled

please reread carefully, the logic is inverted here ;)

Reply:
jaja,  I see now, . Thanks